src/ActiveLogin.Authentication.BankId.Api/BankIdSimulatedApiClient.cs

```diff
@@ -94,7 +95,23 @@ public async Task<AuthResponse> AuthAsync(AuthRequest request)
             return new AuthResponse(orderRef, autoStartToken);
         }
 
-        private string GetPersonalIdentityNumber(AuthRequest request)
+        public async Task<SignResponse> SignAsync(SignRequest request)
```

Comment:
Would like to see a refactoring where the shared logic between AuthAsync and SignAsync are placed in a private method. I think they are identical in what they do, but are working with different input / output.